### PR TITLE
[scatter] allow select country for Europe

### PR DESCRIPTION
**Pull request by chejennifer**
- also populate place type options after place has been selected

For now, the place type options are general to the selected place type which means some of these options may not be real options for the selected place. 

eg.
Screen Shot 2021-11-03 at 6 18 25 PM

### CODE REVIEW

**Comment by pradh** [comment]
Lay audience question:  will we at some point make it like map tool, which seems to know better:

Screen Shot 2021-11-03 at 6 21 56 PM

**Comment by chejennifer** [comment]
> Lay audience question: will we at some point make it like map tool, which seems to know better:
> 
> Screen Shot 2021-11-03 at 6 21 56 PM

I think we definitely should and we could, but scatter is a little harder to tell what child places are possible for a selected place. For example, with Europe, it looks like there actually are stat vars for AdministrativeArea1/2 in Europe
Screen Shot 2021-11-03 at 6 28 42 PM

File: static/css/tools/scatter.scss [hunk 1]
```diff
@@ -140,6 +140,7 @@
 #enclosed-place-type {
   height: 100%;
   font-size: 1rem;
+  width: 225px;
```
Comment (beets) [inline static/css/tools/scatter.scss:143]: nit: for a followup - we could prefer to use a more scalable unit, like rem for this (to accommodate large fonts, etc).

File: static/js/tools/scatter/place_and_type_options.tsx [hunk 2]
```diff
@@ -150,11 +155,12 @@ function PlaceAndTypeOptions(): JSX.Element {
               className="pac-target-input"
             >
               <option value="">Select a place type</option>
-              {childPlaceTypes.map((type) => (
-                <option value={type} key={type}>
-                  {type}
-                </option>
-              ))}
+              {childPlaceTypes &&
```
Comment (beets) [inline static/js/tools/scatter/place_and_type_options.tsx:158]: was there a case you encountered where this list was empty? it seems like it should at least be set to [] by default.

**Review by beets — APPROVED** [review]
thanks for the fix!

File: static/css/tools/scatter.scss (hunk 1 above)
Comment (chejennifer) [reply static/css/tools/scatter.scss:143]: Ok, will do this in a followup

File: static/js/tools/scatter/place_and_type_options.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/tools/scatter/place_and_type_options.tsx:158]: scatter app.test.tsx kept failing complaining about it being undefined .. even though I don't think I've actually ever seen it undefined

**Pull request merged**